examples/rules/README.md

```diff
@@ -0,0 +1,12 @@
+# Rules example
```

Comment:
Instead of adding a rules example, maybe call it custom example and we can use it for creating custom controls and profiles when we add that support later